### ISSUE TITLE
BookmarksBar plugin breaks in Zim 0.66

### ISSUE THREAD

**Issue by vossenjp** [jaap-karssenberg/zim-desktop-wiki#13]
Steps to reproduce:
1. Have Zim 0.66 (from PPA in my case)
1. Enable _BookmarksBar_ in "Edit > Prefs > Plugins"
1. Click on the _BookmarksBar_ icon in the toolbar --or-- "View > Bookmarks" and get:

```
This is zim 0.66
Platform: posix
Locale: en_US UTF-8
FS encoding: UTF-8
Python: (2, 7, 6, 'final', 0)
Gtk: (2, 24, 23)
Pygtk: (2, 24, 0)
No bzr version-info found

======= Traceback =======
  File "/usr/lib/python2.7/dist-packages/zim/actions.py", line 240, in do_activate
    self.__get__(instance, instance.__class__)()
  File "/usr/lib/python2.7/dist-packages/zim/actions.py", line 211, in func
    self.func(instance, active)
  File "/usr/share/zim/plugins/bookmarksbar.py", line 125, in toggle_show_bookmarks
    self.show_widget()
  File "/usr/share/zim/plugins/bookmarksbar.py", line 103, in show_widget
    self.window.add_widget(self.widget, (TOP_PANE, TOP))
AttributeError: 'MainWindowExtension' object has no attribute 'widget'
```
Expected: the _BookmarksBar_ should display instead of showing an exception.  :-)

**Comment by jaap-karssenberg** [comment]
Just tested this, but doesn't reproduce. The line number mentioned in the error is not uptodate with the plugin as shipped in version 0.66. Looks like you have another copy of the plugin installed that overrules the version shipped. Could this be the case?

**Comment by vossenjp** [comment]
Bingo, problem solved.
```
$ locate bookmarksbar
/usr/lib/python2.7/dist-packages/zim/plugins/bookmarksbar.py
/usr/lib/python2.7/dist-packages/zim/plugins/bookmarksbar.pyc
/usr/share/zim/manual/Plugins/BookmarksBar.txt
/usr/share/zim/plugins/bookmarksbar.py

$ sudo rm -v /usr/share/zim/plugins/bookmarksbar.py
[sudo] password for jp: 
removed ‘/usr/share/zim/plugins/bookmarksbar.py’
```
Restart Zim and _BookmarksBar_ works as expected.  Thanks!
